### DISCUSSION TITLE
How to host our events on Github Events Page

### DISCUSSION THREAD

**Discussion by Subramanyarao11** [community/community#40600] [GitHub Education]
We are conducting a hackathon on behalf of our college RNSIT and wanted to promote our event on the Official Github Events Page over [here](https://education.github.com/events). Please do help so that we can utilize this great platform to promote our event and make our event fruitful for event organizers as well as the developers who would be attending the event

**Comment by ossanna16** [comment]
Hi there @Subramanyarao11 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
